src/d3a/models/strategy/__init__.py

```diff
@@ -588,6 +617,14 @@ def can_bid_be_posted(
 
         return total_posted_energy <= required_energy_kWh and bid_price >= 0.0
 
+    def can_settlement_bid_be_posted(self, bid_energy, bid_price, market, replace_existing=False):
```

Comment:
I dont see this method being used somewhere. Can you maybe point me to it? (also `can_settlement_offer_be_posted`)

Reply:
https://github.com/gridsingularity/d3a/pull/1214#discussion_r698711127 , TLDR will be used in the followup story